tests/model_serving/model_server/conftest.py

```diff
@@ -88,6 +88,17 @@ def model_service_account(
         yield sa
 
 
+@pytest.fixture(scope="class")
+def model_service_account_2(admin_client: DynamicClient, models_endpoint_s3_secret: Secret) -> ServiceAccount:
```

Comment:
this is used only in authentication, please move to conftest in that dir

Reply:
ack